source/changelog.txt

```diff
@@ -350,6 +350,12 @@ New features in this release:
 
 - Adds basic support for the ``--apiStrict`` flag.
 
+- New connection methods:
+
+  - :method:`Mongo.getDBNames()` Returns a list of databases.
```

Comment:
[suggestion]

Consider either separating the method name from the method description with a full colon or lowercasing the instances of "Returns" on line 355 and 356.

Reply:
Done.